cli/cmd/cluster-apply.go

```diff
@@ -62,6 +63,8 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 		upgradeKubelets: upgradeKubelets,
 		skipComponents:  skipComponents,
 		verbose:         verbose,
+		configPath:      viper.GetString("lokocfg"),
+		valuesPath:      viper.GetString("lokocfg-vars"),
```

Comment:
Would it not be better if we had a higher level struct which had all the options common to all sub commands ?

These two lines are repetitive for every sub command and will be if another one is added.

Reply:
Probably, yeah. But I didn't want to play around here with deduplication and building some fancy abstraction. My goal was to separate this code from CLI code. It is all definitely not a desired state yet, but moving this around should make further improvements easier.